android-iconify-fontawesome/build.gradle

```diff
@@ -15,6 +15,6 @@ android {
 }
 
 dependencies {
-    implementation "com.android.support:support-annotations:${SUPPORT_LIBRARY_VERSION}"
+    implementation "androidx.annotation:annotation:1.0.0"
```

Comment:
We can use androidX version constant after creation.

Reply:
done!